src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -4,14 +4,20 @@
 
 package net.snowflake.ingest.utils;
 
+import net.snowflake.ingest.SimpleIngestManager;
```

Comment:
We should not import SimpleIngestManager here. I think it happened because of the LOGGER addition. The Logger should use HttpUtil.class